.github/PULL_REQUEST_TEMPLATE/python.md

```diff
@@ -0,0 +1,169 @@
+**Changes:**
+
+* IoT3 SDK:
+    * core: ...
+    * mobility: ...
+* applications:
+    * iqm: ...
+    * ...
+
+Closes: #
+
+---
+**How to test:**
+
+_**Note:** in the following, lines starting with `$` are to be executed on your machine, as a non-root user; lines starting with `(docker)$` are to be executed in the Docker container; lines starting with `(docker)🐍 $` are to be executed in the Docker container, in the python venv._
+
+1. Prepare a test environment:
+    1. be sure to have unrestricted access to _test.mosquitto.org_ (IPv4 and IPv6)
+    2. be sure to have an MQTT broker that listens locally on port 1883, with no credentials and no ACL; if not, run your own:
+        ```sh
+        $ mosquitto
+        ```
+    3. in another terminal, prepare an OpenTelemetry collector on localhost:
```

Comment:
to be precise, you're running a 'jaeger collector' implementing an OTLP interface. e.g, we can (and we ;) ) deploy a "pure" otlp collector. Okay to use the all inone, but just says maybe "a collector implementing the OTLP specifications" : https://opentelemetry.io/docs/collector/ ? Or something like that (and I'll steal your description for Rust :D).

Reply:
OK for `a collector implementing the OTLP specifications`.